<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingPage</class>
 <widget class="QWidget" name="SettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_Menu" native="true">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="tBtn_Menu1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>人员列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tBtn_Menu2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>车间列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tBtn_Menu3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>产线列表</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>471</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>车间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ComB_Department">
           <item>
            <property name="text">
             <string>A1车间</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A2车间</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B2车间</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B3车间</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D1车间</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D2车间</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>线体：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ComB_ProductLine">
           <item>
            <property name="text">
             <string>A1车间</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A2车间</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B2车间</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B3车间</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D1车间</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D2车间</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit">
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2018</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_2">
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2018</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_SearchUser">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="Btn_AddUser">
           <property name="text">
            <string>新增</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_DeleteUser">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_user"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="Btn_addDepartment">
           <property name="text">
            <string>新增</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_DeleteDepartment">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_department"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="Btn_AddProductLine">
           <property name="text">
            <string>新增</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_DeleteProductLine">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_productLine"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
